test/index.js

```diff
@@ -4,7 +4,7 @@ const majorNodeVersion = process.versions.node.split('.')[0];
 
 if (typeof global.gc !== 'function') {
   // Construct the correct (version-dependent) command-line args.
-  let args = ['--expose-gc', '--no-concurrent-array-buffer-freeing'];
+  let args = ['--expose-gc'];
```

Comment:
Did you check the history of why this was added in the first place. I'm wondering if adding it needs to be Node.js version dependent versus just removing completely (ie might results in failures on older Node.js versions...)

Reply:
Sorry, I didn't check it before making this PR. I checked now and the PR that introduced this is: https://github.com/nodejs/node-addon-api/pull/737. It fixed this issue: https://github.com/nodejs/node-addon-api/issues/735. I'm not sure whether removing this flag would end up causing the failure again because we don't have any Windows tests in the CI.

Reply:
I added a check to conditionally add the option if `v8` has a major version that is less than `9`. PTAL.